numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -104,6 +104,10 @@ def simple_lmem(A, B, dty):
         B[i] = C[i]
 
 
+parent_dir = os.path.dirname(os.path.dirname(__file__))
+data_dir = os.path.join(parent_dir, 'data')
```

Comment:
Given this is "spelled" to be at a fixed location in `setup.py` (`numba.cuda.tests.data`), should there be a variable in e.g. `numba.cuda.tests.support` called something like `TEST_DATA_PATH` which is set to this path and then that used throughout the CUDA tests?

Reply:
That would be better, yes.

Reply:
Now added `test_data_path` to `numba.cuda.testing`, which is where lots of utility bits for CUDA tests have been put.